rllib/tuned_examples/dreamerv3/atari_100k.py

```diff
@@ -34,23 +44,21 @@
             "full_action_space": False,
             # Already done by MaxAndSkip wrapper: "action repeat" == 4.
             "frameskip": 1,
-        }
-    )
-    .resources(
-        num_cpus_for_main_process=1,
-    )
-    .learners(
-        num_learners=0 if num_gpus == 1 else num_gpus,
-        num_gpus_per_learner=1 if num_gpus else 0,
+        },
     )
     .env_runners(
+        num_env_runners=(args.num_env_runners or 0),
```

Comment:
Why is it that we use only a single env runner to collect new samples?

Reply:
The training ratio for this setup (Atari 100k) is 1024, which is huge anyways.
You sample 1 step, you update all models on one 1024 sized batch (B=16 x T=64) from the buffer.

So parallelizing the env collection does not make sense. You don't win any performance out of this. DreamerV3 is all about parallelizing on the learner side.